### PR TITLE
Creating a form with Rails 

### PR DESCRIPTION
**Pull request by thneves**
- This an example of how to use forms in Rails, using the Form Helpers.
- The Form is to create a new user and edit a user.
- The form has validations attributes, error messages.

### CODE REVIEW

**Comment by bertil291utn** [comment]
# **Status:** Changes requested 🟠 
<p align="center"> image <br><strong>Not yet</strong> </p>

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitFlow flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:
### General requirements 
- Remove all inline comments line number 7 on app/controllers/user_controllers.rb
### SECTION: edit form
- It should have comment out fom_for 
image
### SECTION: new user
  `form_for`
  - Replace  text input field type input with password input

  `form_tag`
-  Use `label_tag`, `text_field_tag`, `email_field_tag`, `password_field_tag`
### SECTION: edit user
- Replace  text input field type input with password input

### Summary

-  Use right labels on `form _tag` form 
- Replace  text input field type input with password input

***

Guys work on the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
+    @user = User.new(user_params)
```
Comment (bertil291utn) [inline app/controllers/users_controller.rb]: ### Inline comment
- Please remove the inline comment

**Review by bertil291utn — CHANGES_REQUESTED** [review]
# **Status:** Changes requested 🟠 
<p align="center"> image <br><strong>Not yet</strong> </p>

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitFlow flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:
### SECTION: edit form
- It should have comment out fom_for 
image
### SECTION: new user
  `form_for`
  - Replace  text input field type input with password input

  `form_tag`
-  Use `label_tag`, `text_field_tag`, `email_field_tag`, `password_field_tag`
### SECTION: edit user
- Replace  text input field type input with password input

### Summary

-  Use right labels on `form _tag` form 
- Replace  text input field type input with password input

***

Guys work on the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+<input type='submit' value='Submit'>
+
+<%# <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>
+
+<%# 
+<%= form_tag("/users", method: "post") do %>
+  <%# <%= label_tag(:username, "Username:") %>
+  <%# <%= text_field_tag(:username) %>
+  <%# <%= label_tag(:email, "Email:") %> 
+  <%# <%= text_field_tag(:email) %>
```
Comment (ghost) [inline app/views/users/new.html.erb]: change line 23 to `email_field_tag`

**Review by ghost — APPROVED** [review]
# Status: Approved 🎉 

**Team, before you merge, please check out the comment below. Otherwise congratulations  and good luck in your next project 😄.**

File: app/views/users/new.html.erb (hunk 2 above)
Comment (janis-jenny) [reply app/views/users/new.html.erb]: Thanks Rose 🙂

**Pull request merged**